docs/submit/install-ospool-ap.md

```diff
@@ -74,54 +76,69 @@ Initial Steps
+Once your Access Point is registered in OSG Topology and you are registered with COManage,
+you can obtain a token from the [OSG Token Registration page](https://os-registry.opensciencegrid.org/).
+
+You will need a host with Docker to run the software used for retrieving the token.
+Please use your COManage registered and approved identity to log into the
+OSG Token Registration page.
+Follow the instructions on the website; in the list of hosts,
+select the hostname of the Access Point that you registered earlier.
 
+Save this token file; you will use it later when [configuring authentication](#configuring-authentication).
 
 Installing Required Software
 ----------------------------
 Flocking requires HTCondor software as well as software for reporting to the OSG accounting system.
 Start by setting up the EPEL and OSG YUM repositories following the
-[Installing Yum Repositories](../common/yum.md) document. __Note that you have to use OSG 3.6__. Earlier
-versions will not work.
+[Installing Yum Repositories](../common/yum.md) document.
 
-Once the YUM repositories are setup, install the `osg-flock` convenience RPM that installs all
-required packages. Example on a RHEL 7 host:
+Once the Yum repositories are setup, install the `osg-flock` convenience RPM that installs all
+required packages. Example on a RHEL 9 host:
 
 ```console
-# yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-# yum install https://repo.opensciencegrid.org/osg/3.6/osg-3.6-el7-release-latest.rpm
+# yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+# yum install https://repo.opensciencegrid.org/osg/23/osg-23-el9-release-latest.rpm
 # yum install osg-flock
 ```
 
-### Upgrading
+### Upgrading from OSG 3.5 or earlier
+
+The Access Point must be running a supported version of the OSG Software Stack;
+the currently supported versions as of December 2023 are OSG 23 and OSG 3.6.
+If you are running an earlier version, you should upgrade to the most recent version
+of the OSG Software Stack that your operating system supports.
+OSG 23 supports RHEL 8- and 9-compatible operating systems.
+OSG 3.6 supports RHEL 7-, 8-, and 9-compatible operating systems.
+
+See the instructions for upgrading to the appropriate series:
+
+- [Upgrading to the OSG 23 series](https://osg-htc.org/docs/release/updating-to-osg-23/)
+- [Upgrading to the OSG 3.6 series](https://osg-htc.org/docs/release/updating-to-osg-36/)
 
-Upgrading from previous versions should be as simple as switching to OSG 3.6, and then
-issuing `yum upgrade`. If you made local config changes, please verify that the files under
+If you made local config changes, please verify that the files under
 `/etc/condor/config.d` were renamed/disabled during the upgrade.
 
 Note that in some older versions of the package, the Gratia config was kept in 
 `/etc/gratia/condor/ProbeConfig`. The new location is `/etc/gratia/condor-ap/ProbeConfig`.
```

Comment:
We made this move in 3.6 so we should be able to drop this in the PR where you drop <= OSG 3.5 things